charts/nats/values.yaml

```diff
@@ -19,7 +19,7 @@ images:
 
 
 nats:
-  createJetStreamJob: false
+  createJetStreamJob: true
```

Comment:
do we need this flag?

Reply:
this will require clusterAdmin permission since we are using namespace label capabilities.